cirq/work/observable_measurement_test.py

```diff
@@ -198,7 +199,53 @@ def test_aggregate_n_repetitions():
     assert reps == 6
```

Comment:
Could you replace this test with an example using `_check_meas_specs_still_todo`? This would help clarify the purpose of `_aggregate_n_repetitions`.

Reply:
It's impossible with the current two StoppingCriteria because they both either are done or do `repetitions_per_chunk`. If there was a less-well-behaved stopping criteria it could trigger it. Maybe I'll throw one together

Reply:
did a somewhat contrived test :)